-------------------------------------------------------------------------------
Test set: dao.DatabaseTest
-------------------------------------------------------------------------------
Tests run: 10, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 12.261 s <<< FAILURE! - in dao.DatabaseTest
testNoNetwork  Time elapsed: 2.163 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.io.IOException to be thrown, but nothing was thrown.
	at dao.DatabaseTest.testNoNetwork(DatabaseTest.java:100)

